vue/sbc-common-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sbc-common-components",
-  "version": "2.4.54",
+  "version": "2.5.0",
```

Comment:
since it's a node update I suggest starting with a major version update. maybe 3.0.0